### DISCUSSION TITLE
Play time blank

### DISCUSSION THREAD

**Discussion by Mini-Maddie** [vrcx-team/VRCX#982] [Q&A] [answered]
I got this specifically to hopefully check playtime across platforms, but my playtime is just blank. Everything else seems to be working correctly but this was the only reason I downloaded. Is there something I'm missing or is there truly no way to check?

**Comment by BLANKE1** [comment, marked as answer]
VRChat doesn't share playtime of users directly through the readily available json files we already have access to (most likely in their other analytics), so VRCX just locally makes a playtime when you're playing on the same PC whilst VRCX is open.

**Reply by Mini-Maddie** [reply]
Ok thank you!
